MicrosoftQuickAuthSwiftSupport.podspec

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |s|
+  s.name                  = 'MicrosoftQuickAuthSwiftSupport'
+  s.version               = '0.0.5'
```

Comment:
we can start from 0.0.1

Reply:
Resolved by changing the version to `0.0.1`.